### ISSUE TITLE
Opus Diff only somewhat helpful.

### ISSUE THREAD

**Issue by jeffturcotte** [imarc/opus#8]
If I make changes to a file and opus detects an integrity issue when updating, the diff provided to me is the full current version of the installed file compared to the new incoming file. While this diff should always be provided, I typically also want to see a diff only of what I changed since I added the file, which would provide far more insight into what I would need to port over if I choose to overwrite.

I understand that this is probably a job for the vcs and not opus, but (with a single terminal) I cannot get this information inline during an update and feel like I'm flying blind with files like CSS where the full diff can be very hard to parse. It might actually be possible with Opus if the diffs of all installed assets/files are calculated pre-update, but that's probably crazy.

We should discuss some potential solutions here.

**Comment by mattsah** [comment]
I'm wrong, you can't reference the old one.  It would be possible to diff the old ones prior to update, however, and store those diffs for viewing.  If this was done, it may also be possible to attempt 3 way merges which would bring in both the additions/changes from the new stuff then merge your changes on top, although that could get messy with conflicts.  Let's discuss at codejam if you're still interested in this feature.
